doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst

```diff
@@ -255,24 +255,24 @@ Applications must enable these options if they generate factory data or need an
 
 .. _child_parent_to_sysbuild_migration_nrf700x:
 
-nRF700x
-=======
+the nRF70 Series
+================
 
-Support for nRF700x operating mode and firmware storage has moved to sysbuild.
+Support for the nRF70 Series operating mode and firmware storage has moved to sysbuild.
 The following Kconfig options are available:
 
 +----------------------------------------------------------------+-----------------------------------------------------------------------------+
 | Kconfig option                                                 | Description                                                                 |
 +================================================================+=============================================================================+
-|               ``SB_CONFIG_WIFI_NRF700X``                       | Enable Wifi support for nRF700x                                             |
+|               ``SB_CONFIG_WIFI_NRF70``                         | Enable Wifi support for nRF700x                                             |
 +----------------------------------------------------------------+-----------------------------------------------------------------------------+
-|               ``SB_CONFIG_WIFI_NRF700X_SYSTEM_MODE``           | Use system mode firmware patches and set application to this mode           |
+|               ``SB_CONFIG_WIFI_NRF70_SYSTEM_MODE``             | Use system mode firmware patches and set application to this mode           |
 +----------------------------------------------------------------+-----------------------------------------------------------------------------+
-|               ``SB_CONFIG_WIFI_NRF700X_SCAN_ONLY``             | Use scan-only mode firmware patches and set application to this mode        |
+|               ``SB_CONFIG_WIFI_NRF70_SCAN_ONLY``               | Use scan-only mode firmware patches and set application to this mode        |
 +----------------------------------------------------------------+-----------------------------------------------------------------------------+
-|               ``SB_CONFIG_WIFI_NRF700X_RADIO_TEST``            | Use radio test mode firmware patches and set application to this mode       |
+|               ``SB_CONFIG_WIFI_NRF70_RADIO_TEST``              | Use radio test mode firmware patches and set application to this mode       |
 +----------------------------------------------------------------+-----------------------------------------------------------------------------+
-|               ``SB_CONFIG_WIFI_NRF700X_SYSTEM_WITH_RAW_MODES`` | Use system with raw modes firmware patches and set application to this mode |
+|               ``SB_CONFIG_WIFI_NRF70_SYSTEM_WITH_RAW_MODES``   | Use system with raw modes firmware patches and set application to this mode |
```

Comment:
```suggestion
|               ``SB_CONFIG_WIFI_NRF70_SYSTEM_WITH_RAW_MODES``   | Use system with Raw modes firmware patches and set application to this mode |
```

Reply:
Can you please correct the mention of **nRF7000x** in line 307 which is incorrect?